Tryouts/Core/Services/ModulesService/MorganStanley.ComposeUI.Tryouts.Core.Services.ModulesService.csproj

```diff
@@ -28,10 +28,4 @@ See the License for the specific language governing permissions and limitations
     <ProjectReference Include="..\..\..\Core\Abstractions\MorganStanley.ComposeUI.Tryouts.Core.Abstractions.csproj" />
     <ProjectReference Include="..\..\..\Core\Utilities\MorganStanley.ComposeUI.Tryouts.Core.Utilities.csproj" />
   </ItemGroup>
-
-  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-  	<Exec Command="xcopy &quot;$(OutDir)\$(AssemblyName).dll&quot; &quot;$(SolutionDir)\bin\$(Configuration)\net6.0\Plugins\Services\$(ProjectName)\&quot; /R /Y /I" />
-  	<Exec Command="xcopy &quot;$(OutDir)\$(AssemblyName).pdb&quot; &quot;$(SolutionDir)\bin\$(Configuration)\net6.0\Plugins\Services\$(ProjectName)\&quot; /R /Y /I" />
-  </Target>
-
```

Comment:
I agree with removing these, it seems like a bad idea to copy specifically like this, but make sure it doesn't break other people's work without discussing with them

Reply:
These was the left out code from a very brief time when the Modules implementations was a dynamically loaded plugin. It is not needed anymore since the Modules are now statically loaded. 
No other projects will be affected - tested it on my own machine.